ui/components/logs/empty-state.tsx

```
@@ -0,0 +1,250 @@
+			</div>
+
+			{isSocketConnected && (
+				<div className="flex items-center justify-center gap-2 text-sm">
+					<RefreshCw className="h-4 w-4 animate-spin" />
+					<span>Listening for logs...</span>
+				</div>
+			)}
+
+			<div className="grid w-full max-w-4xl grid-cols-1 gap-6 px-4 md:grid-cols-2">
+				{CARDS.map((card) => (
+					<Card className="p-6">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)